wallet/wallet.go

```diff
@@ -3788,6 +3807,48 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 			backendError: err,
 		}
 
+	// Returned by bitcoind when a transaction does not meet the fee
+	// requirements to be accepted into mempool. This happens when the
+	// mempool reached its limits and is now purging low fee transactions.
+	// Increasing the mempool size limit (maxmempool) will solve this
+	// problem so we log the appropriate bitcoind config setting.
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L510
+	case match(err, "mempool min fee not met"):
+		returnErr = &ErrMempoolFee{
+			backendError: err,
+		}
+		log.Infof("%v: tx not accepted into mempool because of low "+
+			"fees, consider raising your bitcoind maxmempool "+
+			"config value", txid)
+
+	// Returned by btcd when a transaction does not meet the fee
+	// requirements to be accepted into mempool.
+	// https://github.com/btcsuite/btcd/blob/9c16d23918b15c468c5647c388b9b7db3bc48dc7/mempool/mempool.go#L1151
+	case match(err, "fees which is under the required amount"):
+		fallthrough
+
+	// Returned by btcd when a transaction does not meet the fee
+	// requirements to be accepted into mempool. The error states priority
+	// but decreasing the min relay fee prevents checking for the priority
+	// in the first place therefore we consider this a mempool fee error.
+	// https://github.com/btcsuite/btcd/blob/9c16d23918b15c468c5647c388b9b7db3bc48dc7/mempool/mempool.go#L1162
+	case match(err, "has insufficient priority"):
+		fallthrough
+
+	// Returned by bitcoind when a transaction does not meet the fee
+	// requirements to be accepted into mempool because the policy of the
+	// mempool has a higher min relay fee. Default value for bitcoind is
+	// 1000 sat/kvbyte but this is configurable so we log the appropriate
+	// bitcoind config setting.
+	// https://github.com/bitcoin/bitcoin/blob/9bf5768dd628b3a7c30dd42b5ed477a92c4d3540/src/validation.cpp#L514
+	case match(err, "min relay fee not met"):
+		returnErr = &ErrMempoolFee{
+			backendError: err,
+		}
+		log.Infof("%v: tx not accepted into mempool because of low "+
```

Comment:
think this log is a bit redundant since we now have this explicit error.

Reply:
to me the error will be handled by the caller - either update the fee or just log it, so seems unnecessary.

Reply:
sounds good removed both cases of the logging.